bb-integration-tests/src/main/java/com/cognifide/qa/bb/analytics/TestModule.java

```diff
@@ -0,0 +1,32 @@
+/*-
+ * #%L
+ * Bobcat
+ * %%
+ * Copyright (C) 2019 Cognifide Ltd.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.cognifide.qa.bb.analytics;
+
+import com.cognifide.qa.bb.modules.CoreModule;
+import com.google.inject.AbstractModule;
+
+public class TestModule extends AbstractModule {

```

Comment:
Let's call this `AnalyticsTestModule` - we have too many test modules flying around already :)